next.config.js

```diff
@@ -16,6 +16,12 @@ module.exports = {
         destination: "https://discord.gg/WHpCrPqeqx",
         permanent: true,
       },
+      {
+        source: "/linkedin-page",
```

Comment:
Love this idea. Shall we tweak to be just `linkedin`? Similar specificity as "Discord" versus something like "Discord invite"

Reply:
yup makes sense, made the adjustment in the next commit